hack/run-upstream-argo-rollouts-e2e-tests.sh

```diff
@@ -25,7 +25,12 @@ cd argo-rollouts
 git checkout $CURRENT_ROLLOUTS_VERSION
 go mod tidy
 
-# 2) Setup the Namespace
+# 2) Replace 'argoproj/rollouts-demo' image with 'quay.io/jgwest-redhat/rollouts-demo' in upstream E2E tests
+# - The original 'argoproj/rollouts-demo' repository only has amd64 images, thus some of the E2E tests will not work on Power/Z.
+# - 'quay.io/jgwest-redhat/rollouts-demo' is based on the same code, but built for other archs
+find "$TMP_DIR/argo-rollouts/test/e2e" -type f | xargs sed -i.bak  's/argoproj\/rollouts-demo/quay.io\/jgwest-redhat\/rollouts-demo/g'
```

Comment:
LGTM, but is it possible to use team's repo?

Reply:
Thanks Jayendra, good question! Because this project is within argoproj-labs, we would likely need to use the [argoproj-labs quay.io org](https://quay.io/organization/argoprojlabs). I will follow up on that as part of a separate PR. (But obviously anywhere is better than my personal repo :smile: )